bin/lib/cli.js

```diff
@@ -21,7 +22,7 @@ function getVersion () {
     return stdout.trim()
   } catch (e) {
     // Obtain version from package.json
-    const { version } = require('../package.json')
+    const { version } = require(path.join(__dirname, '../../package.json'))
```

Comment:
Minor tip:  did you know you can do 

path.join(__dirname, '..', '..', 'package.json')

instead of

path.join(__dirname, '../../package.json')